foreman/data_refinery_foreman/foreman/management/commands/track_data_request.py

```diff
@@ -0,0 +1,133 @@
+            samples_count = experiment.samples.count()
+            samples_attempted += samples_count
+            samples_attempted_by_source.update({source: samples_count})
+
+            # Experiments available, total and breakdown by source.
+            processed_samples_count = experiment.samples.filter(is_processed=True).count()
+            if processed_samples_count > 0:
+                experiments_available += 1
+                experiments_available_by_source.update({source: 1})
+
+                # Samples available, total and breakdown by source.
+                samples_available += processed_samples_count
+                samples_available_by_source.update({source: processed_samples_count})
+
+            sample_accessions = experiment.samples.values_list("accession_code", flat=True)
+            # Total number of jobs created, breakdown by source.
+            downloader_jobs_created = self.get_downloader_jobs(sample_accessions).count()
+            processor_jobs_created = self.get_processor_jobs(sample_accessions).count()
+            survey_jobs_created = self.get_surveyor_jobs(experiment).count()
+            total_jobs_created = (
+                downloader_jobs_created + processor_jobs_created + survey_jobs_created
+            )
+
+            jobs_created += total_jobs_created
+            jobs_created_by_source.update({source: total_jobs_created})
+
+            # TODO(arkid15r): Calculate total run time as a difference between first created job
+            # and the last finished job. Also indicate if there is something that still needs to
+            # be processed.
+
+        output = []
+        if experiments_attempted:
+            output += [
+                f"Experiments attempted: {experiments_attempted}",
+                f"{self.get_distribution_by_source(experiments_attempted_by_source)}",
+                "",
+            ]
+
+            output.append(f"Samples attempted: {samples_attempted}")
+            if samples_attempted:
+                output.append(f"{self.get_distribution_by_source(samples_attempted_by_source)}")
+            output.append("")
+
+            output.append(f"Experiments available: {experiments_available}")
+            if experiments_available:
+                output.append(f"{self.get_distribution_by_source(experiments_available_by_source)}")
+            output.append("")
+
+            output.append(f"Samples available: {samples_available}")
+            if samples_available:
+                output.append(f"{self.get_distribution_by_source(samples_available_by_source)}")
+            output.append("")
+
+            output.append(f"Total jobs: {total_jobs_created}")
+        else:
+            output.append("No experiments found")
+
+        print("\n".join(output))
+
+    @staticmethod
+    def get_accessions(url):
+        """Generates source experiment accessions."""
+        return (line.strip() for line in requests.get(url).text.split("\n") if line.strip())
```

Comment:
Could we test for protocol and then use `boto3` as done here: https://github.com/AlexsLemonade/refinebio/blob/dbc64643b43b3b8e86a63e91d10a5ddfb043ca2e/foreman/data_refinery_foreman/surveyor/management/commands/unsurvey.py#L242-L254

Reply:
Done.